app/config/locale/templates/email-base.tpl

```diff
@@ -1,67 +1,92 @@
 <!doctype html>
 <html>
+    <head>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&family=Poppins:wght@500;600&display=swap">
+        <style>
+            body {
+                padding: 32px;
+                line-height: 1.5;
+                color: #616b7c;
+                font-size: 15px;
+                font-weight: 400;
```

Comment:
why do we have so many changes in the email templates?

Reply:
I made email-base and email-base-styled to have as much in-common as possible. Thats why those changes. Now the files are pretty much copy&paste, except one extra `<style>`, footer and header.

Reply:
I decide to do it when I was fixing like third problem caused by those inconsistencies - wrong word breaking, missing background color, paddings misbehaving.. There were some fixed made to one fine and not to the other one, and it was hard to apply properly since these files were much different

Reply:
Okay my main concern is, that when we fetch them on cloud through the vendor directory, it should not break any of the billing emails 🤔

Reply:
It might, but then those styles are meant to be in cloud anyway. Im pretty pretty sure cloud has it's own email because, because open source didn't have footer until now. So this should not be breaking.

Reply:
After latest merge with 1.5.x, this seems to b ea bit less changes